src/deepsparse/pipeline.py

```diff
@@ -404,7 +406,11 @@ def __call__(self, *args, **kwargs):
 
         kwargs["inference_state"] = inference_state
 
-        return self.run(*args, **kwargs)
+        next_call = self.run
+        if self.middleware_manager is not None:
+            next_call = self.middleware_manager.build_middleware_stack(next_call)
```

Comment:
nit: could we rename to something like `wrapped_call`? I feel that's closer to what this variable actually holds?